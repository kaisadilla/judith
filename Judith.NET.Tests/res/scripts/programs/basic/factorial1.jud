print("Factorial of 5: ")
__p_print factorial(5)

print("Factorial of 2: ")
__p_print factorial(2)

print("Factorial of 10: ")
__p_print factorial(10)

func factorial (num: F64) : F64
    if num == 1 then return 1 end

    return num * factorial(num - 1)
end