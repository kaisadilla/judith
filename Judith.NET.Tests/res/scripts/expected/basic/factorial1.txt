Factorial of 5: 120
Factorial of 2: 2
Factorial of 10: 3.6288e+06
